contracts/JBSplitsStore.sol

```diff
@@ -183,10 +208,77 @@ contract JBSplitsStore is IJBSplitsStore, JBOperatable {
         revert INVALID_TOTAL_PERCENT();
       }
 
-      // Push the new split into the project's list of splits.
-      _splitsOf[_projectId][_domain][_group].push(_splits[_i]);
+      uint256 _packedSplitParts1 = _splits[_i].preferClaimed ? 1 : 0;
+      _packedSplitParts1 |= _splits[_i].percent << 1;
```

Comment:
we could probably use an upper bounds check of an individual split's percent above

Reply:
isnt the check for the total percents enough?

Reply:
true dat